app/models/music_brainz_client.rb

```diff
@@ -0,0 +1,14 @@
+class MusicBrainzClient
+
+  def self.request_artist_data(artist)
+    url = "http://musicbrainz.org/ws/2/artist/?query=name:#{artist}&fmt=json"
+    @response = Faraday.get(url)
+  end
+
+  def self.parse_data
+    data = JSON.parse(@response.body)
+    artist = data["artists"][0]
+    artist_data = artist.slice("name")
+    artist_data.merge!("city": artist.dig('begin-area', 'name'), "country": artist.dig('area', 'name'), "year-started": artist.dig('life-span', 'begin'), "year-ended": artist.dig('life-span', 'end'))
```

Comment:
nice work. this line (12) might be more difficult than we need. Its impressive mind you

we can instead create a hash and only stuff values from the JSON response that we need into it. That way we can see the hash that is being built. Just something to consider 
e.g. 

```
artist_hash = {
   name: data[:something][0]
   city: data[:something_else]
   etc
```